include/zephyr/bluetooth/hci_types.h

```diff
@@ -34,6 +34,8 @@ extern "C" {
 #define BT_HCI_H4_ISO                   0x05    /* HCI ISO Data packet */
 
 /* Special own address types for LL privacy (used in adv & scan parameters) */
```

Comment:
Change/Move comment as the newly added defines are not special, only the later are special due to the reason of being using only in adv & scan parameters.